src/System.Private.CoreLib/shared/System/Version.cs

```diff
@@ -433,14 +430,8 @@ private static bool TryParseComponent(ReadOnlySpan<char> component, string compo
             return (v1.CompareTo(v2) <= 0);
```

Comment:
not related to your change, but why `<=` instead of `<` ?

Reply:
Because this is the `<=` operator. Using `<` would be wrong.